server/simple/bin/server.dart

```diff
@@ -7,24 +7,41 @@ import 'dart:io';
 import 'package:shelf/shelf.dart';
 import 'package:shelf/shelf_io.dart';
 import 'package:shelf_router/shelf_router.dart';
+import 'package:shelf_static/shelf_static.dart';
 
 Future main() async {
   // Find port to listen on from environment variable.
   // https://cloud.google.com/run/docs/reference/container-contract#port
   final port = int.parse(Platform.environment['PORT'] ?? '8080');
 
+  // Serve static files from a file system directory.
+  final _staticHandler = createStaticHandler('public');
+
+  // Cascade helps in chaining multiple handlers together.
+  final _cascade = Cascade().add(_staticHandler).add(_router);
+
   // Serve handler on given port.
   final server = await serve(
-    _router,
+    _cascade.handler,
     InternetAddress.anyIPv4, // Allows external connections
     port,
   );
+
   print('Serving at http://${server.address.host}:${server.port}');
 }
 
+// Router instance to handler requests.
 final _router = Router()
   ..get('/', _helloWorldHandler)
+  ..get('/dashland', _dashLandHandler)
   ..get('/time',
       (request) => Response.ok(DateTime.now().toUtc().toIso8601String()));
 
 Response _helloWorldHandler(Request request) => Response.ok('Hello, World!');
+
+Response _dashLandHandler(Request request) =>
```

Comment:
Added the handler to make it serve via route instead of serving the HTML directly. Although, does this makes the example complex?

Reply:
We shouldn't be doing this, right? This is duplicating the shelf_static logic, right?

Reply:
Okay I'll remove the logic, but shelf serves the file not via a route I guess ? 

So if I person wants to show content on a particular route, this is what they should be  doing right ?

Reply:
Or you mean the part where I'm referencing the file, taking its content again and sending as a response ?

Reply:
maybe just serve static content from the root if nothing else is accessed?